### PR TITLE
Set Initial Viewpoint added

### PR DESCRIPTION
**Pull request by harish-kunchala**
Set initial Viewpoint added

### CODE REVIEW

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                  ),
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
```
Comment (PaulAllanSturm) [inline lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:52]: For super-simple samples like this one, we get rid of `_ready` and this progress indicator (see e.g. add_tiled_layer.dart)

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import 'package:flutter/material.dart';
+
+class SetInitialViewpoint extends StatefulWidget {
+  const SetInitialViewpoint({super.key});
+
+  @override
+  State<SetInitialViewpoint> createState() => _SetInitialViewpointState();
+}
+
+class _SetInitialViewpointState extends State<SetInitialViewpoint> {
```
Comment (PaulAllanSturm) [inline lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:26]: I think we still want `with SampleStateSupport` and the import

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    final initialEnvelope = Envelope.fromXY(
+      xMin: -118.805,
+      yMin: 34.027,
+      xMax: -118.795,
+      yMax: 34.037,
+      spatialReference: SpatialReference.wgs84,
+    );
+    // Create and set the initial viewpoint to the map
+    final initialViewPoint = Viewpoint.fromTargetExtent(initialEnvelope.extent);
+    map.initialViewpoint = initialViewPoint;
```
Comment (PaulAllanSturm) [inline lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:79]: I would just combine these two lines and get rid of the `initialViewPoint` variable

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+    // Create an initial envelope.
+    final initialEnvelope = Envelope.fromXY(
+      xMin: -118.805,
+      yMin: 34.027,
+      xMax: -118.795,
+      yMax: 34.037,
+      spatialReference: SpatialReference.wgs84,
+    );
+    // Create and set the initial viewpoint to the map
```
Comment (PaulAllanSturm) [inline lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:77]: comments should have a period at the end

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void onMapViewReady() {
```
Comment (PaulAllanSturm) [inline lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:67]: the customary comment is
```dart
  // Create a map with a topographic basemap style.
```

Comment (harish-kunchala) [reply lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:67]: Added the comment.

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart (hunk 4 above)
Comment (harish-kunchala) [reply lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:77]: Added the period.

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart (hunk 3 above)
Comment (harish-kunchala) [reply lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:79]: Removed the `initialViewPoint` variable.

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart (hunk 2 above)
Comment (harish-kunchala) [reply lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:26]: Added the `SampleStateSupport` import.

File: lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart (hunk 1 above)
Comment (harish-kunchala) [reply lib/samples/set_initial_viewpoint/set_initial_viewpoint.dart:52]: Yeah removed the `Visibility` , `_ready` and made my app look similar to `add_tiled_layer.dart`.

**Pull request merged**